doc/ref_model/chapters/chapter01.md

```diff
@@ -334,7 +335,7 @@ As with most CNTT subsystems, responsibility for Objectives, Requirements, Guide
 <a name="1.10"></a>
 ## 1.10 Roadmap
 The NFVI Reference Model and Reference Architecture will continue to be refined as technology and industry needs change over time.  Release 1 of this document will focus on the network function virtualisation infrastructure and virtualised network functions which are based on virtual machines. In Release 2 there will be partial support for cloud native functions which make use of container technology.  
-The first reference architecture is based on Openstack, but the intention is to expand the portfolio of Reference Architectures with an upcoming focus in release 2 to areas such as Containerization, Kubernetes-based Cloud Native stacks and Container based network functions’ validation requirement.  Other planned additions to the project in future releases include support for:
+The first reference architecture is based on OpenStack, but the intention is to expand the portfolio of Reference Architectures with an upcoming focus in release 2 to areas such as Containerization, Kubernetes-based Cloud Native stacks and Container based network functions’ validation requirement.  Other planned additions to the project in future releases include support for:
```

Comment:
Should we delete this or replace this paragraph without stating first, second etc.

Reply:
I have no preference on this, but I would not like to do bigger changes on this paragraph in this pr.